client/src/components/VerifyInfo/index.tsx

```diff
@@ -19,7 +19,11 @@ const styles = (theme: Theme) =>
     },
   });
 
-function VerifyInfo({ classes }: WithStyles<typeof styles>) {
+type Props = WithStyles<typeof styles> & {
+  onVerify: () => void;
```

Comment:
Will `onVerify` need to be async to query the backend?

Reply:
Yessir. Will update in next PR